### ISSUE TITLE
Translation to German language to ....

### ISSUE THREAD

**Issue by WebCrew** [RomainOdeval/batflat#11]
Translation to **German language** to give the CMS more reach.

**I`ll do it** so you can **assigne me** to the job if you want @RomainOdeval 

Kind regards, WebCrew ;)

**Comment by WebCrew** [comment]
I updated the CMS and the Docu and some of the more important modules with de_german language files. The rest of the modules will be also done at a later time. One two wekks or so. 

The docu in particular was a lot of work, I hadn't even thought of it when I said I'd do the translations lol.

**Comment by RomainOdeval** [comment]
Partially resolved with #13

Need language ini files for some modules

**Comment by RomainOdeval** [comment]
> Sorry I'm a bit confused right now as I'm doing this translation and at the same time a translation for Vvveb CMS, both using .po and .mo files. Is the translation for Barflat now complete or is something still missing?

Yeah, maybe one day i will switch to po/mo or yaml files but not a priority :( 
I think the main doc is incomplete and some modules have not their german files.
